bots/run-queue

```diff
@@ -52,6 +52,8 @@ def consume_webhook_queue(channel, amqp):
         sha = request['sha']
         context = pipes.quote(request['context'])
         cmd = ['bots/tests-scan', '--sha', sha, '--amqp', amqp, '--context', context]
+    elif event == 'issues':
+        cmd = ['bots/issue-scan', '--amqp', amqp]
```

Comment:
We need to check more boxes in our webhook for that, right? That ought to be safe to do right now already, as unknown events should just be ignored.

Reply:
yup, just the issues event though